components/rollup_node.md

```diff
@@ -0,0 +1,81 @@
+# Rollup Node
```

Comment:
Is `Rollup Node` more canonical than `client`? Both terms seem good to me, I just don't have the context for the evolution of eth2 terminology so I'm curious curious

Reply:
I tend to use `node` and `client`mostly  interchangeably. Maybe the distinction to me would be a node is a "thing" while a client is the actual "software"?